robot/.pathplanner/settings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this being used? We can remove it if not.

Reply:
I don't know if we'll want it for the center notes, but I'd guess not. I'm still working on cleaning up some stuff, and I'll look at that next